samples/nrf9160/modem_shell/src/gnss/gnss.h

```diff
@@ -173,6 +227,57 @@ int gnss_set_nmea_mask(uint16_t nmea_mask);
  */
 int gnss_set_priority_time_windows(bool value);
 
+/**
+ * @brief Sets the dynamics mode.
+ *
+ * @param mode Dynamics mode.
+ *
+ * @retval 0 if the operation was successful.
+ *         Otherwise, a (negative) error code is returned.
+ */
+int gnss_set_dynamics_mode(enum gnss_dynamics_mode mode);
+
+/**
+ * @brief Sets the QZSS NMEA mode.
```

Comment:
I agree that the comments could be more verbose, but I was a bit lazy and though that it would be enough that the possible values are documented for the enum:
```
enum gnss_qzss_nmea_mode {
	/** Standard NMEA PRN reporting. */
	GNSS_QZSS_NMEA_MODE_STANDARD,
	/** Custom NMEA PRN reporting (PRN IDs 193...202 used for QZSS satellites). */
	GNSS_QZSS_NMEA_MODE_CUSTOM
};
```

Reply:
Added details about the modes to the function description.